packages/video_player_avplayer/tizen/src/video_player_tizen_plugin.cc

```
@@ -0,0 +1,340 @@
+      if (iter != drm_configs->end()) {
+        if (std::holds_alternative<std::string>(iter->second)) {
+          license_server_url = std::get<std::string>(iter->second);
+        }
+      }
+    }
+
+    const flutter::EncodableMap *player_options = msg.player_options();
+    if (player_options) {
+      auto iter =
+          player_options->find(flutter::EncodableValue("prebufferMode"));
+      if (iter != player_options->end()) {
+        if (std::holds_alternative<bool>(iter->second)) {
+          prebuffer_mode = std::get<bool>(iter->second);
+        }
+      }
+    }
+  } else {
+    return FlutterError("Invalid argument", "Either asset or uri must be set.");
+  }
+
+  int64_t player_id = 0;
+  if (uri.substr(0, 4) == "http") {
+    auto player = std::make_unique<PlusPlayer>(plugin_registrar_->messenger(),
+                                               native_window, format);
+    player_id =
+        player->Create(uri, drm_type, license_server_url, prebuffer_mode);
+    players_[player_id] = std::move(player);
+  } else {
+    auto player = std::make_unique<MediaPlayer>(plugin_registrar_->messenger(),
+                                                native_window);
+    player_id =
+        player->Create(uri, drm_type, license_server_url, prebuffer_mode);
+    players_[player_id] = std::move(player);
+  }
+
+  PlayerMessage result(player_id);
+  return result;
+}
+
+std::optional<FlutterError> VideoPlayerTizenPlugin::Dispose(
+    const PlayerMessage &msg) {
+  auto iter = players_.find(msg.player_id());
+  if (iter != players_.end()) {
+    iter->second->Dispose();
+    players_.erase(iter);
+  }
+  return std::nullopt;
+}
+
+std::optional<FlutterError> VideoPlayerTizenPlugin::SetLooping(
+    const LoopingMessage &msg) {
+  VideoPlayer *player = FindPlayerById(msg.player_id());
+  if (!player) {
+    return FlutterError("Invalid argument", "Player not found.");
+  }
+  player->SetLooping(msg.is_looping());
```

The operation of player may be failed, it should not always return 'std::nullopt'.
Please throw an error in player when operation failed, use 'try' and 'catch' to get the error.
I think all of player operations should process error.

---

Fixed.